sfs/plot.py

```diff
@@ -284,6 +285,9 @@ def soundfield(p, grid, xnorm=None, cmap='coolwarm_clip', vmin=-2.0, vmax=2.0,
     if ax is None:
         ax = plt.gca()
 
+    if matplotlib_version.startswith('2.1.'):
+        p = np.clip(p, -1e15, 1e15)  # clip to float64 range
```

Comment:
Maybe we add a link to the matplotlib bug (https://github.com/matplotlib/matplotlib/issues/10567) on the line before to highlight why we are doing this?

Reply:
Otherwise this implementation is fine with me.